addons/binding/org.openhab.binding.elerotransmitterstick/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="elerotransmitterstick" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I thought about your modelling of things again and I do not like the way you are handling the elero-channelIDs...

I propose that you change your model as follows:

- have one bridge which is the stick with parameters `portName` and `updateInterval` (i.e. keep the one you have)
- only have ONE thing-type `eleroThing` which has:
-- a parameter (in a config-description) `channelIDs` which is set as "multiple" (see [here](https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#xml-structure-for-configuration-descriptions)) containing the values 0 to 15.
-- a channel group with your 2 channels `control` and `status`
- drop the thing-type "elerochannel"

This is much cleaner than your solution because it has the relevant elero-channel-ID as a thing parameter and not in an array index or thing property where it doesn't belong. Also it saves you from parsing a comma separated string of channelIDs for your group because the parameter will be handed over to you as a list.

Reply:
I just converted the channel ids in the group to a integer parameter of type multiple. This does not work at all, either I am doing something wrong or the implementation in the paper ui is buggy.
I create a group and select channel ids (which are sorted by their string representation, which is not optimal) 1 and 10 . I apply and edit the group again. If I now deselect channel id 1, the string shows "1,10,1". There is no way to get rid of the selected values any more. This does not seem to be usable this way.

Reply:
I am not sure I agree with you about having only one thing type. A single channel can not have multiple ids, this just does not make sense. I see your point in using a multiple parameter of type integer, but as long as this does not work in the paper UI I see no point in changing it. I can also change the channel id to be a parameter, but i decided to store it as a property because it does not make sense to change it. If I set it as readOnly, can it then only be specified on creation?

Reply:
I have now made the channel ID a read-only integer configuration parameter and updated the README to contain example configuration. That should be it, then.

Once the multiple parameter works in the paper ui i will update the group thing to use it.

Reply:
ChannelId as a parameter is better than having it a s a property, thank you for changing!

I still think that both thing types can be replaced by one that accepts just one channel id. As you have documented in the code in https://github.com/openhab/openhab2-addons/pull/2659/files#diff-6906ae24e01be461fa3856be014ea1eeR229 : 

> "this is a workaround for a bug in the stick firmware that does not handle commands that are sent to multiple channels correctly"

you are sending the command to each channel, one by one. Isn't there a group address that you can use as a "channel id"? Because as you have it implemented at the moment it would be better to have the grouping done by an openHAB group (containing multiple elerochannel things, each having one elerochannel-Id) and not as a separate thing-type.